services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -42,6 +45,20 @@
     get_services_client,
 )
 
+
+class AttachNetworkToDynamicSidecarItem(BaseModel):
+    project_id: ProjectID
+    node_id: NodeID
+    network_name: DockerNetworkName
+    network_alias: DockerNetworkAlias
+
+
+class DetachNetworkFromDynamicSidecarItem(BaseModel):
```

Comment:
I guess these models should not be in this module right?

Reply:
Yes, they are just used for the API Requests

Reply:
Move these to `osparc-simcore/services/director-v2/src/simcore_service_director_v2/models/domains/dynamic_services.py`